.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
I think, if we omit this, it will run on _all_ PR's, rather than just those to `main`, which is probably what we want?

```suggestion
```